tests/unit/lms/services/file_test.py

```diff
@@ -7,24 +7,39 @@
 from tests import factories
 
 
-class TestGet:
-    def test_it_returns_the_matching_file_if_there_is_one(
+class TestFileService:
```

Comment:
Named after a method but testing the whole class. Updated the naming of the class and methods.

Reply:
:+1: Good catch